source/delegate/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@airswap/delegate",
-  "version": "0.5.6",
+  "version": "0.5.6-beta.0",
```

Comment:
if you release a beta, is it normally the number before or the number after? i.e. if you have version 1 and then you will have version 2 in future, and you have a beta inbetween, should it be version 2 beta?

Reply:
Beta in my mind wasn't related to product version. Still trying to work out how beta relates to the rest of the release process but wanted to unblock integration work.